OTBR integration: usually doesn't need to be installed manually (#30948)
home-assistant/home-assistant.github.io: source/_integrations/otbr.markdown

---

The Open Thread Border Router integration allows calling an Open Thread Border Router's REST API from Python and via WebSocket.

> This integration is installed automatically when the **Open Thread Border Router** add-on is installed. This happens when you do one of the following procedures:

Reviewer: The add-on can also be used in combination with other hardware. I'd adjust it to something like this.

```suggestion
This integration is installed automatically when the **Open Thread Border Router** add-on is installed. For  Home Assistant Yellow or SkyConnect refer to the following procedures:
```

- [Enabling Thread on Home Assistant Yellow](https://yellow.home-assistant.io/procedures/enable-thread/)
- [Enabling Thread on Home Assistant SkyConnect](https://skyconnect.home-assistant.io/procedures/enable-thread/)

> 
> The integration is installed automatically when the **Silicon Labs Multiprotocol** add-on is installed. This happens when you do one of the following procedures:
> 
> - [Enabling Multiprotocol on Home Assistant Yellow](https://yellow.home-assistant.io/guides/enable-multiprotocol/)
> - [Enabling Multiprotocol on Home Assistant SkyConnect](https://skyconnect.home-assistant.io/procedures/enable-multiprotocol/)

Reviewer: Since the dedicated Thread is the promoted solution for now, let's drop the links here.

```suggestion

The integration is also installed automatically when enabling the experimental **Silicon Labs Multiprotocol** support.
```

---

The Open Thread Border Router integration allows calling an Open Thread Border Router's REST API from Python and via WebSocket.

This integration is installed automatically when the **Open Thread Border Router** add-on is installed. This happens when you do one of the following procedures:

- [Enabling Thread on Home Assistant Yellow](https://yellow.home-assistant.io/procedures/enable-thread/)
- [Enabling Thread on Home Assistant SkyConnect](https://skyconnect.home-assistant.io/procedures/enable-thread/)

> - [Adding a HomeKit device via Thread](https://www.home-assistant.io/integrations/homekit_controller/)
> - [Adding a Matter device via Thread](https://www.home-assistant.io/integrations/matter/#adding-a-matter-device-to-home-assistant)

Reviewer: When adding Thread devices via 3rdparty border routers the OTBR integration won't be used.

Just remove these two, the main gist is that this happens when installing the OTBR add-on.

```suggestion
```
